src/elektra/redshift.ini

```diff
@@ -0,0 +1,234 @@
+; Elektra specification file for redshift@490ba2aae9cfee097a88b6e2be98aeb1ce990050 (= redshift v1.12 + some small commits)
+; This file is based on: https://github.com/ElektraInitiative/libelektra/blob/master/examples/spec/lcdproc/LCDd.ini
+;
+; A spec file "(...) defines how the configuration settings should be validated" (source: https://www.libelektra.org/tutorials/validate-configuration)
+; For details see: https://www.libelektra.org/tutorials/writing-a-specification-for-your-configuration)
+
+; Redshift uses no environment variables.
+
+[]
+mountpoint = redshift.ecf
+
+; Configuration values - section redshift
```

Comment:
Why not make a real section or even a separate config file?

Reply:
Good point thanks. Actually some of the comments - including this one - where outdated, I fixed that.